source/engine/ecs/Transform.cpp

```diff
@@ -58,7 +58,9 @@ namespace nc
 
     void Transform::SetRotation(const Quaternion& quat)
     {
-        DirectX::XMVECTOR scl_v, rot_v, pos_v;
+        DirectX::XMVECTOR pos_v = DirectX::XMVectorZero();
```

Comment:
Should we ignore the "might be uninitialized" error instead of default initializing these?